src/app/globals.css

```
@@ -113,17 +126,65 @@
 }
 
-/* Grid pattern background */
+/* Liquid-glass top navigation pill. The SVG filter (#nav-liquid-glass)
+   refracts the backdrop at the pill's rim; plain blur is the fallback for
+   browsers without SVG backdrop-filter support. */
+.liquid-glass-nav {
+  @apply relative overflow-hidden rounded-full border border-transparent bg-transparent shadow-none;
+  transition:
+    background 0.26s ease,
+    border-color 0.26s ease,
+    box-shadow 0.26s ease;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)